.github/workflows/beam_PreCommit_Go_Commit.yml

```
@@ -0,0 +1,34 @@
+name: beam_PreCommit_Go_Commit
```

```suggestion
name: beam_PreCommit_Go
```

Since this will encapsulate both precommit, postcommit, and probably eventually trigger jobs lets go with a more general name. Same applies to the file name and job name

---

I see, sure
Job, file names were updated
Thanks